_security/authentication-backends/saml.md

```diff
@@ -19,37 +19,37 @@ This profile is meant for use with web browsers. It is not a general-purpose way
 
 We provide a fully functional example that can help you understand how to use SAML with OpenSearch Dashboards.
 
-1. Download [the example zip file]({{site.url}}{{site.baseurl}}/assets/examples/saml-example-custom.zip) to a preferred location in your directory and unzip it.
-1. At the command line, specify the location of the files in your directory and run `docker-compose up`.
-1. Review the files:
-
-   * `customize-docker-compose.yml`: Defines two OpenSearch nodes, an OpenSearch Dashboards server, and a SAML server.  
-   * `customize-opensearch_dashboards.yml`: Includes SAML settings for the default `opensearch_dashboards.yml` file.
-   * `customize-config.yml`: Configures SAML for authentication.
-
-   You can remove "customize" from the file names if you plan to modify and keep these files for production.
-   {: .tip }  
-
-1. In the `docker-compose.yml` file, specify your OpenSearch version number in the `image` field for nodes 1 and 2 and the OpenSearch Dashboards server. For example, if you are running OpenSearch version {{site.opensearch_major_minor_version}}, the `image` fields will resemble the following examples:
-   
-   ```yml
-   opensearch-saml-node1:
-    image: opensearchproject/opensearch:{{site.opensearch_major_minor_version}}
-    ```
-    ```yml
-    opensearch-saml-node2:
-    image: opensearchproject/opensearch:{{site.opensearch_major_minor_version}}
-    ```
-    ```yml
-    opensearch-saml-dashboards:
-    image: opensearchproject/opensearch-dashboards:{{site.opensearch_major_minor_version}}
-    ```
-
-1. Access OpenSearch Dashboards at [http://localhost:5601](http://localhost:5601){:target='\_blank'}. Note that OpenSearch Dashboards immediately redirects you to the SAML login page.
-
-1. Log in to OpenSearch Dashboards. The default username is `admin` and the default password is set in your `customize-docker-compose.yml` file in the `OPENSEARCH_INITIAL_ADMIN_PASSWORD=<custom-admin-password>` setting.
-
-1. After logging in, note that your user in the upper-right is `SAMLAdmin`, as defined in `/var/www/simplesamlphp/config/authsources.php` of the SAML server.
+Note: These steps require basic knowledge about interacting with GitHub. If you are new to GitHub then check-out this [guide](https://github.com/opensearch-project/demos/blob/main/ONBOARDING.md) to get started.
```

Comment:
It's only the first step that requires knowledge of GitHub, right? Can we remove this note and add something to Step 1, like "1. Download the [saml-demo](https://github.com/opensearch-project/demos/tree/saml-demo) branch on the [demos](https://github.com/opensearch-project/demos) repository to a preferred location in your directory. If you are new to GitHub, see [guide](https://github.com/opensearch-project/demos/blob/main/ONBOARDING.md)." Do we need the second ink to [demos]?

Reply:
second link is not needed. I've addressed all outstanding feedback.